core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletOperationId.java

```diff
@@ -33,14 +33,14 @@ public class TabletOperationId extends AbstractId<TabletOperationId> {
 
   public static String validate(String opid) {
     var fields = opid.split(":");
-    Preconditions.checkArgument(fields.length == 2, "Malformed operation id %s", opid);
+    Preconditions.checkArgument(fields.length == 4, "Malformed operation id %s", opid);
```

Comment:
Could use the following String method then, can leave the rest of the code as is (like check for 2 and assume the field[1] is the entire fate id even if it has ":")

```java
var fields = opid.split(":",2);
```